performance_test/src/communication_abstractions/resource_manager.cpp

```diff
@@ -60,12 +65,12 @@ eprosima::fastrtps::Participant * ResourceManager::fastrtps_participant() const
   eprosima::fastrtps::xmlparser::XMLProfileManager::getDefaultParticipantAttributes(PParam);
   PParam.rtps.sendSocketBufferSize = 1048576;
   PParam.rtps.listenSocketBufferSize = 4194304;
-  PParam.rtps.builtin.use_SIMPLE_RTPSParticipantDiscoveryProtocol = true;
-  PParam.rtps.builtin.use_SIMPLE_EndpointDiscoveryProtocol = true;
-  PParam.rtps.builtin.m_simpleEDP.use_PublicationReaderANDSubscriptionWriter = true;
-  PParam.rtps.builtin.m_simpleEDP.use_PublicationWriterANDSubscriptionReader = true;
+  eprosima::fastrtps::rtps::DiscoverySettings & disc_config = PParam.rtps.builtin.discovery_config;
+  disc_config.use_SIMPLE_EndpointDiscoveryProtocol = true;
+  disc_config.m_simpleEDP.use_PublicationReaderANDSubscriptionWriter = true;
+  disc_config.m_simpleEDP.use_PublicationWriterANDSubscriptionReader = true;
+  disc_config.leaseDuration = eprosima::fastrtps::c_TimeInfinite;
```

Comment:
@BMarchi Are these changes compatible with FastRTPS version 1.9 or later?

Reply:
It seems they are. I ran perf tests with fastrtps and it works fine